src/arch/generic/BaseSemihosting.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2018, 2019 Riscv Limited
```

Comment:
This code is basically copied/extracted from ArmSemihosting.py so it should bear the original copyright holder (Arm Limited), whereas here you replaced Arm with Riscv

Reply:
I failed to exclude this line from my search+replace :( Should be fixed now.

Reply:
It seems like this was fixed in the wrong commit.